json-schema.gemspec

```
@@ -16,13 +15,11 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ['README.md', 'LICENSE.md']
   s.required_ruby_version = '>= 2.7'
   s.license = 'MIT'
-  s.required_rubygems_version = '>= 2.5'
 
-  s.add_development_dependency 'bundler'
   s.add_development_dependency 'minitest', '~> 5.0'
-  s.add_development_dependency 'rake'
+  s.add_development_dependency 'rake', '~> 13.2', '>= 13.2.1'
```

Why so specific? I'd imagine this works just as well
```suggestion
  s.add_development_dependency 'rake', '~> 13.0'
```